src/scrabble.js

```diff
@@ -1,5 +1,55 @@
-function scrabble() {
-  // write code here
+function scrabble(word) {
+  if (!word || typeof word !== 'string' || word.trim() === '') return 0
+
+  const scores = {
```

Comment:
this can go outside the function; since it is constant variable that is read-only..

you can also rename it as `SCRABBLE_SCORES` or `SCRABBLE_LETTER_SCORES` to indicate that it is read only and be more specific about what it represents

Reply:
all noted.